language-extensions/index.d.ts

```diff
@@ -1,2 +1,56 @@
 declare function $multi<T extends any[]>(...values: T): MultiReturn<T>;
 declare type MultiReturn<T extends any[]> = T & { readonly " __multiBrand": unique symbol };
+
```

Comment:
Some nitpicking on the "operator" naming:

I would prefer complete names without abbreviations, I think that would increase readability and since you don't have to type these Identifiers very often, it shouldn't result in a lot of extra type work for the user. You could call them the same as in `ExtensionKind` (https://github.com/TypeScriptToLua/TypeScriptToLua/pull/969/files#diff-32793566a7017d38a85af44995a5400e56cab8d4894d8d545fd586496312d878R4)

e.g `LuaFloorDivision` instead of `LuaIDiv`
`LuaBitwiseLeftShift` instead of `LuaShl` ...

Reply:
My thought here was that someone writing these declarations would know the meta-method they were mapping to and be looking for that. I don't feel strongly about it though - I can rename them.

Reply:
I think for most people this will be the first time interacting with Lua operators.  I for example had trouble figuring out what `unm` is supposed to mean, `UnaryNegation` on the other hand I would have understand immediately.